randovania/games/samus_returns/logic_database/Area 6.txt

```diff
@@ -1229,13 +1225,9 @@ Extra - asset_id: collision_camera_Hazard_End_A
   > Pickup (Super Missile Tank)
       All of the following:
           # Completing the puzzle
-          Grapple Beam and Morph Ball and Screw Attack and Shoot Any Missile
+          Grapple Beam and Morph Ball and Power Bomb ≥ 3 and Screw Attack and Shoot Any Missile
           # Traversing the crumble blocks
           Phase Drift or Spider Ball or Movement (Beginner)
-          All of the following:
-              # Breaking the Bomb blocks
-              Lay Power Bomb
-              Bomb or Power Bomb ≥ 3
```

Comment:
The bomb requirement is gone here?

Reply:
Same as the above comment. Having Bombs saves one power bomb. No need to account for two items to save one ammo